src/axom/sidre/CMakeLists.txt

```diff
@@ -105,6 +110,7 @@ set(sidre_depends
 
 blt_list_append(TO sidre_depends ELEMENTS hdf5 IF HDF5_FOUND)
 blt_list_append(TO sidre_depends ELEMENTS sparsehash IF SPARSEHASH_FOUND)
+blt_list_append(TO sidre_depends ELEMENTS mfem IF MFEM_FOUND AND AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION)
```

Comment:
> MFEM_FOUND AND AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION

@white238  -- I thought these compound expressions don't work in this macro.
I am checking that this is doing the right thing, and will post what I find out.

Reply:
I think this line has to change...

I just configured with:
```
>./config-build.py -hc host-configs/rzgenie-toss_3_x86_64_ib-gcc@8.1.0.cmake -DAXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION=OFF
```
and mfem was included in sidre's dependencies.  I.e. `blt_print_target_properties(TARGET mfem)` included:
```
-- [sidre property] BLT_SIDRE_DEPENDS_ON: core;conduit::conduit;slic;hdf5;sparsehash;mfem;fmt;mpi
                                                                                     ~~~~
```

I'll fix this.

Reply:
Update: Fixed.